doc/source/conf.py

```diff
@@ -247,7 +249,9 @@
         "plausible_analytics_url": "https://views.scientific-python.org/js/script.js",
     },
     "logo": {"image_dark": "https://pandas.pydata.org/static/img/pandas_white.svg"},
+    "navbar_align": "left",
     "navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],
+    "sidebar_includehidden": False,
```

Comment:
Did you add this for a specific reason? (locally I am not directly seeing a difference)

Reply:
Ah I thought this was needed before because the empty sidebar misaligned the body of the page, but looks like this isn't needed anymore